wordpress-mobile/WordPress-iOS
# Blogging Prompts: Add `BloggingPromptSettings` & `BloggingPromptSettingsReminderDays`

See: #18549

## Description

Adds the Core Data models for blogging prompt settings and the reminder days for it.

## Testing

The `fetchSettings` call needs to be added manually. I added it to `DashboardPromptsCardCell`:
```swift
    func skipMenuTapped() {
//        saveSkippedPromptForSite()
//        presenterViewController?.reloadCardsLocally()
        bloggingPromptsService?.fetchSettings(success: {}, failure: { _ in })
    }
```

To test:
- Add code to fetch prompt settings
- Launch app and navigate to wherever it was added and make the network call
- Verify the model properly maps from the remote model


## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
